plasmapy/formulary/__init__.py

```diff
@@ -5,18 +5,18 @@
 # __all__ will be auto populated below
 __all__ = []
 
-from .braginskii import *
-from .collisions import *
-from .dielectric import *
-from .dimensionless import *
-from .distribution import *
-from .drifts import *
-from .ionization import *
-from .magnetostatics import *
-from .mathematics import *
-from .parameters import *
-from .quantum import *
-from .relativity import *
+from .braginskii import *  # noqa: F403
+from .collisions import *  # noqa: F403
+from .dielectric import *  # noqa: F403
+from .dimensionless import *  # noqa: F403
+from .distribution import *  # noqa: F403
+from .drifts import *  # noqa: F403
+from .ionization import *  # noqa: F403
+from .magnetostatics import *  # noqa: F403
+from .mathematics import *  # noqa: F403
+from .parameters import *  # noqa: F403
+from .quantum import *  # noqa: F403
+from .relativity import *  # noqa: F403
```

Comment:
Yes, star imports are a terrible anti pattern, yes, this file is a little bit of a special case... I think we could however make this a little cleaner by using https://stackoverflow.com/a/54454433/4417567. So we could replace this entire thing by adding
```
per-file-ignores =
    plasmapy/formulary/__init__.py:F403
```
to the flake8 section of setup.cfg. How does that sound?